lib/iopromise/data_loader.rb

```diff
@@ -23,7 +30,7 @@ def self.included(base)
 
     def data_promises
```

Comment:
**Naming** Related to the naming suggestion above (`async_attr`), this method could then perhaps be called `async_attributes`?